node/cmd/guardiand/admintemplate.go

```diff
@@ -932,6 +962,100 @@ func runWormholeRelayerSetDefaultDeliveryProviderTemplate(cmd *cobra.Command, ar
 	fmt.Print(string(b))
 }
 
+func runGeneralPurposeGovernanceEvmCallTemplate(cmd *cobra.Command, args []string) {
+	if *governanceTargetAddress == "" {
+		log.Fatal("--target-address must be specified")
+	}
+	if !common.IsHexAddress(*governanceTargetAddress) {
+		log.Fatal("invalid target address")
+	}
+	governanceTargetAddress := common.HexToAddress(*governanceTargetAddress).Hex()
```

Comment:
Why are we not using `parseAddress` in these templates?

Reply:
because `parseAddress` attempts to parse any 32 byte address-like thing into a (left-padded) 32 byte address https://github.com/wormhole-foundation/wormhole/blob/51d34bf4a389ad2251a0d4e053ee1c1448baa703/node/cmd/guardiand/admintemplate.go#L1061-L1084

in this case, we only care about EVM addresses, so we can be a bit more specific